### ISSUE TITLE
Alfred fails to recognize keyword upon typing the last letter?

### ISSUE THREAD

**Issue by jngnyc** [swissmanu/search-jira-issues-alfred-workflow#10]
*tl;dr* : Alfred fails to recognize keyword upon typing the last letter. This one is a weird one.

Attached is a zipped screen recording of this odd behavior.
[jira-alfred-search.mov.zip](https://github.com/swissmanu/search-jira-issues-alfred-workflow/files/106690/jira-alfred-search.mov.zip)
Second by second synopsis of the screen recorded behavior:
1. From 0s - 5s, I type the keyword slowly, letter by letter; 'j' 'i' 'r' - observe that Alfred recognizes that I am typing the 'jira' keyword
2. At 5s, I type the last letter of the keyword; 'a' - observe that Alfred suddenly forgets that I am typing the 'jira' keyword. 
3. At 7s, I press backspace to erase the 'a' leaving me with 'jir', - observe that Alfred suddenly remembers I am typing the 'jira' keyword. 
4. at 8s, with 'jir', I press 'enter' to try and force Alfred to use the 'jira' keyword/workflow - observe that then dutifully completes the 'jira' keyword (from me pressing 'enter') but then forgets that I am actually trying to USE the 'jira' keyword.

Screenshot of my Alfred config:
![jira-alfred-search-config](https://cloud.githubusercontent.com/assets/2365139/12616205/da2deafc-c4d7-11e5-89d4-27f40559ac9d.png)

Things I've tried:
- I've followed the installation instructions to a T. I'm seeing this behavior out of the box, haven't even modify the config.sample.json yet. 

- changing the keyword - I've tried changing the keyword from 'jira' to a slew of different things (eg: testing, abcd, helpmeplease, etc). Doesn't seem to be keyword related b/c whatever I try the same behavior occurs; upon typing the last letter of the keyword causes Alfred to forget that 

Am I doing something completely wrong?  Thanks in advance folks.

**Comment by jngnyc** [comment]
This looks the same as [issue 7](https://github.com/swissmanu/search-jira-issues-alfred-workflow/issues/7) and [issue 10](https://github.com/swissmanu/search-jira-issues-alfred-workflow/issues/8)
